src/index.js

```diff
@@ -11,24 +11,9 @@ export {SEPARATOR} from './exporter';
 
 export default function(deckName) {
   const exporter = new Exporter(getDb(), getZip());
-  const { topDeckId, topModelId } = exporter;
 
-  exporter.db.run(getTemplate());
-
-  exporter
-    .updateInitialDecksWith({
-      name: deckName,
-      top_deck_id: topDeckId
-    })
-    .updateInitialModelsWith({
-      name: deckName,
-      css: getCssTemplate(),
-      did: topDeckId,
-      id: topModelId,
-    });
-
-  return ['addCard', 'addMedia', 'save'].reduce((prev, i) => {
-    prev[i] = exporter[i];
-    return prev;
-  }, {});
+  return exporter
```

Comment:
Why not like I suggested in https://github.com/ewnd9/anki-apkg-export/pull/7#issuecomment-263780460 ? `deckName` may still be passed as a first arguments (to prevent breaking changes) and the other options as second.